Yup, no love.

@danp Are the following both expected in the travis logs?

* `Caused by: org.eclipse.jgit.errors.MissingObjectException: Missing commit 18708eda27327fc4acec1f53d2b4571baa7fd0fc`
* `[ConfigurationServiceClientFileImpl.java:138] : Failed to get configuration out of /var/lib/oba/config.json, continuing without it.
java.io.FileNotFoundException: File '/var/lib/oba/config.json' does not exist`

--- a/Dockerfile
+++ b/Dockerfile
@@ -3,7 +3,7 @@
 FROM maven:3.5.2-jdk-8 as build
 
 RUN git clone --depth 1 https://github.com/OneBusAway/onebusaway-application-modules.git /app
-RUN cd /app && mvn package -Dlicense.skip=true -Dmaven.test.skip=true --quiet
+RUN cd /app && mvn install -Dlicense.skip=true -Dmaven.test.skip=true --quiet
 
 FROM tomcat:8
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -3,7 +3,7 @@
 FROM maven:3.5.2-jdk-8 as build
 
 RUN git clone --depth 1 https://github.com/OneBusAway/onebusaway-application-modules.git /app
-RUN cd /app && mvn install -Dlicense.skip=true -Dmaven.test.skip=true --quiet
+RUN cd /app && mvn package -Dlicense.skip=true -Dmaven.test.skip=true --quiet
 
 FROM tomcat:8
 

--- a/config/onebusaway-api-webapp-data-sources.xml
+++ b/config/onebusaway-api-webapp-data-sources.xml
@@ -7,19 +7,17 @@
   </bean>
   <bean id="apiKeyValidationService" class="org.onebusaway.users.impl.validation.KeyValidationServiceImpl" />
   <!-- Database Configuration -->
-  <bean id="dataSource" class="org.springframework.jndi.JndiObjectFactoryBean">
-    <property name="jndiName" value="java:comp/env/jdbc/appDB" />
-    <property name="lookupOnStartup" value="true" />
-    <property name="cache" value="true" />
-    <property name="proxyInterface" value="javax.sql.DataSource" />
-    <property name="resourceRef" value="true" />
+  <bean id="dataSource" class="org.springframework.jdbc.datasource.DriverManagerDataSource">
+    <property name="driverClassName" value="org.hsqldb.jdbcDriver" />
+    <property name="url" value="jdbc:hsqldb:file:/bundle/org_onebusaway_transit_data" />
+    <property name="username" value="sa" />
+    <property name="password" value="" />
   </bean>
   <bean id="agencyMetadataDataSource" class="org.springframework.jndi.JndiObjectFactoryBean">
-    <property name="jndiName" value="java:comp/env/jdbc/appDB" />
-    <property name="lookupOnStartup" value="true" />
-    <property name="cache" value="true" />
-    <property name="proxyInterface" value="javax.sql.DataSource" />
-    <property name="resourceRef" value="true" />
+    <property name="driverClassName" value="org.hsqldb.jdbcDriver" />
+    <property name="url" value="jdbc:hsqldb:file:/bundle/org_onebusaway_transit_data" />
+    <property name="username" value="sa" />
+    <property name="password" value="" />
   </bean>
   <bean id="agencyMetadataSessionFactory" class="org.springframework.orm.hibernate3.annotation.AnnotationSessionFactoryBean">
     <property name="dataSource" ref="agencyMetadataDataSource" />

